packages/cli/scripts/build.ts

```diff
@@ -1,8 +1,12 @@
 import cpy from 'cpy';
 import execa from 'execa';
+import fs from 'fs-extra';
 import { join } from 'path';
-import { remove, writeFile } from 'fs-extra';
+import { fileURLToPath } from 'url';
```

Comment:
Do we want to adopt `node:` for all Node.js internal modules? That seems to be the futureproof pattern.

`node:` was introduced in Node 12 and since we support Node >=14, we should be able to safely switch. We would need to update `@types/node`, which you've bumped it already to v14.